cf-agent/verify_files.c

```diff
@@ -912,6 +933,14 @@ PromiseResult ScheduleEditOperation(EvalContext *ctx, char *filename, const Attr
 
     if (a->haveeditline)
     {
+        if (!fileexists)
+        {
+            RecordFailure(ctx, pp, a,
+                          "Promised to edit '%s', but file does not exist",
+                          filename);
+            result = PromiseResultUpdate(result, PROMISE_RESULT_FAIL);
+        }
+
```

Comment:
This part seems related to edit line promises, not templates?

Reply:
We removed this check from before this function call, we therefore need to add it inside the function for edit line promises to keep working the way they did before.